radis/lbl/base.py

```diff
@@ -2337,13 +2682,9 @@ def _calc_populations_noneq_multiTvib(
             )
 
             #            df.loc[idx, 'Qvib'] = Qvib
-            df.loc[idx, "Q"] = Q
+            df.loc[df.index.tolist(), "Q"] = Q
```

Comment:
same; we shouldn't had "Q" anymore. 

[and if we had to, using `df.loc[:, "Q"]` would be better